### PR TITLE
Adds platformId to login event

### PR DESCRIPTION
**Pull request by TrekSoft**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -15,8 +18,22 @@
...
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "target": "es5",
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true
```
Comment (roscioli) [inline tsconfig.json:29]: Curious, did u mean to add these?

**Review by roscioli — APPROVED** [review]
one question but loox good otherwise!
unnamed

File: tsconfig.json (hunk 1 above)
Comment (TrekSoft) [reply tsconfig.json:29]: I didn't intentionally add them but reviewing the changes that apparently my IDE automatically added they seemed good.

Comment (roscioli) [reply tsconfig.json:29]: i don't think we want `skipLibCheck: true` cuz that means we won't type check our type files :/

Comment (TrekSoft) [reply tsconfig.json:29]: Ah, ok - I'll remove that then

Comment (roscioli) [reply tsconfig.json:29]: the only thing i'm iffy on other than that is `noEmit`... not exactly sure what it does-- don't wanna hold this PR up too much tho

Comment (roscioli) [reply tsconfig.json:29]: cuz apparently it means that no files are outputted? so its only used to typecheck, not compile, typescript... wondering if our build will still work correctly in that case...

Comment (roscioli) [reply tsconfig.json:29]: https://stackoverflow.com/questions/55002137/typescript-noemit-use-case

**Pull request merged**